src/mrpro/operators/models/TransientSteadyStateWithPreparation.py

```diff
@@ -45,34 +45,44 @@ class TransientSteadyStateWithPreparation(SignalModel[torch.Tensor, torch.Tensor
     def __init__(
         self,
         sampling_time: float | torch.Tensor,
-        repetition_time: float,
-        m0_scaling_preparation: float = 1.0,
-        delay_after_preparation: float = 0.0,
+        repetition_time: float | torch.Tensor,
+        m0_scaling_preparation: float | torch.Tensor = 1.0,
+        delay_after_preparation: float | torch.Tensor = 0.0,
     ):
         """Initialize transient steady state signal model.
 
+        `repetition_time`, `m0_scaling_preparation` and `delay_after_preparation` can vary for each voxel and will be
+        broadcasted starting from the front (i.e. from the other dimension).
+
         Parameters
         ----------
         sampling_time
-            time points when model is evaluated. A sampling_time of 0 describes the first acquired data point after the
+            Time points when model is evaluated. A sampling_time of 0 describes the first acquired data point after the
             inversion pulse and spoiler gradients. To take the T1 relaxation during the delay between inversion pulse
             and start of data acquisition into account, set the delay_after_preparation > 0.
             with shape (time, ...)
         repetition_time
             repetition time
         m0_scaling_preparation
-            scaling of the equilibrium magnetisation due to the preparation pulse before the data acquisition
+            Scaling of the equilibrium magnetisation due to the preparation pulse before the data acquisition.
```

Comment:
We use magnetization (AE) in all the example files. Maybe we want to adjust this everywhere in `src/mrpro/operators/models/TransientSteadyStateWithPreparation.py`